### ISSUE TITLE
Update docs on modifying performance settings

### ISSUE THREAD

**Issue by metasoarous** [system76/docs#590]
The docs here seem to be out of date, as the latest version of Pop!_OS no longer has these performance settings in the GNOME Desktop system menu:

https://support.system76.com/articles/graphics-switch-pop/

**Comment by jacobgkau** [comment]
The performance profiles are still listed in the top-right menu in the latest version of Pop!_OS (20.10):

![Screenshot from 2021-04-30 09-31-35](https://user-images.githubusercontent.com/7199422/116718219-f83f0400-a996-11eb-9151-08a0ec1917ad.png)

As mentioned in the article, these are added by gnome-shell-extension-system76-power: https://github.com/pop-os/gnome-shell-extension-system76-power 

The graphics switching modes are also still present on machines with switchable graphics:

![Screenshot from 2021-04-30 09-54-33](https://user-images.githubusercontent.com/7199422/116721040-1c501480-a99a-11eb-9581-851013ed272b.png)

I don't see anything else in that article that you could be referring to, but if you still believe something is out-of-date, please specify and we can re-open this issue.

---

System76 customers can reach out to [support](https://support.system76.com/) for technical assistance. For non-System76 hardware, you can seek community support on [Reddit](https://reddit.com/r/pop_os/) or [Mattermost](https://chat.pop-os.org).

**Comment by metasoarous** [comment]
Hi @jacobgkau. Thanks for the response.

I installed Pop!_OS 20.04 and saw these settings, but they're no longer there after updating to 20.10, so this caught me off guard. To clarify, the additional shell extension is now required for this functionality, but was not previously?

Thanks again

**Comment by jacobgkau** [comment]
> To clarify, the additional shell extension is now required for this functionality, but was not previously?

No, the shell extension has always been required to display the options in the GUI menu. It's what adds the options to the menu.

The functionality can also be accessed via the CLI using the `system76-power` command, which is also [documented in the article](https://support.system76.com/articles/graphics-switch-pop/#from-the-command-line). This should still work without the shell extension.

> I installed Pop!_OS 20.04 and saw these settings, but they're no longer there after updating to 20.10, so this caught me off guard.

This is not a support forum, but I would recommend confirming that both `system76-power` and `gnome-shell-extension-system76-power` are installed (they are dependencies of `pop-session`, so they should be), that the `system76-power` service is enabled and is starting successfully, and that the GNOME shell extension is enabled in the Extensions app (green icon).

---

System76 customers can reach out to [support](https://support.system76.com/) for technical assistance. For non-System76 hardware, you can seek community support on [Reddit](https://reddit.com/r/pop_os/) or [Mattermost](https://chat.pop-os.org).

**Comment by metasoarous** [comment]
> This is not a support forum

Understood; Didn't assume as much. Frankly, I'm not at all attached to the menu items; I usually have a terminal open, and am happy to run the `system76-power` command, which I found described in the docs.

It's still unclear to me why when I first installed Pop!_OS, the menu options were available (I certainly didn't manually install the shell extension), but not after the upgrade to 20.10, so I thought that something had changed in the base install, and that this change could be better reflected in the docs. Only trying to be helpful here, so that others have an easier time taking advantage of Pop!_OS (which I love!).

If none of this makes sense, or if you don't feel anything is amiss in the docs, feel free to ignore.

Thanks for your work
